package.json

```diff
@@ -39,7 +39,8 @@
   },
   "dependencies": {
     "classnames": "^2.5.1",
-    "flatpickr": "^4.6.13"
+    "flatpickr": "^4.6.13",
+    "react-modal": "^3.16.1"
```

Comment:
@scottdickerson @constantinehuzenko I am fine with using react-modal since it is pretty widely used but since we were looking at [this](https://www.radix-ui.com/primitives) library for primitives do we want to use this library to build our modal instead?